src/backend/wiki_game_info/protondb/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { ProtonDBInfo, ProtonDBCompatibilityInfo } from 'common/types'
+import { ProtonDBCompatibilityInfo } from 'common/types'
```

Comment:
I don't like the name `ProtonDBCompatibilityInfo` for me `ProtonDBInfo` fits much more.
Just my opinion.